main.go

```diff
@@ -43,59 +43,76 @@ func init() {
 	// Initializes all commands
 	handler := cmd.New()
 	rootCmd.AddCommand(&cobra.Command{
-		Use:   "login",
-		Short: "Login to Railway",
-		RunE:  contextualize(handler.Login),
+		Use:               "login",
+		Short:             "Login to Railway",
+		PersistentPreRunE: contextualize(handler.CheckVersion),
```

Comment:
Instead of doing this as a persistent pre run, can we not just do it on line 44 once? Also was gonna say check version doesn't need to be part of the handler but I think it does.

Sigh, we might need to refactor this CLI architecture to something more similar to what Stripe and GH are doing